CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+- Enforce utf-8 for reading the readme in setup.py [#67](https://github.com/danielgtaylor/python-betterproto/pull/67)
+- Add intentionally failing test-cases for unimplemented bug-fixes [#55](https://github.com/danielgtaylor/python-betterproto/pull/55)
+- Only import types from grpclib when type checking [#52](https://github.com/danielgtaylor/python-betterproto/pull/52)
+- Re-organized test cases [#51](https://github.com/danielgtaylor/python-betterproto/pull/51)
```

Comment:
Not sure about the value of putting non-functional updates in the change log.

Reply:
Agreed :-) Refactorings and small internal improvements we could leave out

Reply:
Sounds good to me, feel free to bundle or exclude any that aren't particularly relevant to users